### PR TITLE
WORK: CLI tests for images with data disks and capture.

### PR DESCRIPTION
**Pull request by pramodgms**
The content to be added to Changelog is as follows:
* 
* 
*

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@pramodgms__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (pramodg). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: test/commands/arm/diskvm/arm.diskvm-parameter-create-tests.js [hunk 1]
```diff
@@ -238,23 +244,31 @@ describe('arm', function() {
...
+						  var cmd = makeCommandStr('managed-image', 'data-disks', 'delete', imgParamFileName, util.format('--managed-disk --snapshot --index 0')).split(' ');
+                            testUtils.executeCommand(suite, retry, cmd, function(result) {
+                            result.exitStatus.should.equal(0);
+                              var cmd = util.format('managed-image create -g %s -n %s --parameter-file %s --json', groupName, imgPrefix6, imgParamFileName).split(' ');
+                              testUtils.executeCommand(suite, retry, cmd, function(result) {
+                              result.exitStatus.should.equal(0);
+                              done();
+							});
+					      });
+					    });
```
Comment (huangpf) [inline test/commands/arm/diskvm/arm.diskvm-parameter-create-tests.js]: Please align the lines.

File: test/commands/arm/diskvm/arm.diskvm-parameter-create-tests.js [hunk 2]
```diff
@@ -283,6 +297,55 @@ describe('arm', function() {
         });
       });
       
+	  it('vm deallocate and generalize command should pass', function(done) {
+        this.timeout(vmTest.timeoutLarge * 20);
+        var cmd = util.format('vm deallocate --resource-group %s --name %s --json', groupName, vm1Prefix).split(' ');
+        testUtils.executeCommand(suite, retry, cmd, function(result) {
+          result.exitStatus.should.equal(0);
+		  var cmd = util.format('vm generalize --resource-group %s --name %s --json', groupName, vm1Prefix).split(' ');
```
Comment (huangpf) [inline test/commands/arm/diskvm/arm.diskvm-parameter-create-tests.js]: line alignment...

File: test/commands/arm/diskvm/arm.diskvm-parameter-create-tests.js [hunk 3]
```diff
@@ -33,8 +33,11 @@ var requiredEnvironment = [{
 }];
 
 var groupName,
+  vmId,
+  subscriptionId,	
   avsPrefix6 = 'xplattestavs6',
   imgPrefix6 = 'xplattestimg6',
```
Comment (huangpf) [inline test/commands/arm/diskvm/arm.diskvm-parameter-create-tests.js]: There is still reference to this param and the test is complaining:
https://travis-ci.org/Azure/azure-xplat-cli/builds/199443239

**Comment by huangpf** [comment]
closing this one, as it's included in: https://github.com/Azure/azure-xplat-cli/pull/3482

**Pull request closed without merging**